### DISCUSSION TITLE
[ Bug ] - My MAS screen stuck in Birthday screen

### DISCUSSION THREAD

**Discussion by BMVP1234** [Monika-After-Story/MonikaModDev#9595] [QA - tech support + general] [answered]
![image](https://user-images.githubusercontent.com/104214224/191938371-46bfdfea-6c58-4e71-ac6a-e06a8bd986d7.png)

When it was her birthday,I said :"I will take you somewhere",she generated a file and closed the game.When I re-opened the game,the screen was stucked.Can anyone help me with this ?

**Comment by Booplicate** [comment, marked as answer]
Is her file still in `characters/`? If you moved it somewhere, just return.
Otherwise [restore a backup](https://github.com/Monika-After-Story/MonikaModDev/wiki/FAQ#i-need-to-restore-an-older-persistent).
